src/static/js/base.js

```diff
@@ -296,6 +277,19 @@ window.addEventListener("load", function() {
   var asideSelector = ".toc-sidebar";
   if (!document.querySelector(asideSelector)) return;
 
-  var g = new GSAside(asideSelector, ".base-content", 60);
+  var g = new GSAside(asideSelector, ".base-content", 100);
   g.init();
 });
+
+
+/* Deal with clicks on nav links that do not change the current anchor link. */
+$("a" ).click(function() {
```

Comment:
I don't think we do, but in my defense I was modifying something that already existed in this case. (Though the diff is not very clear on that since I moved the whole block)

Reply:
The selector changed, you need to refactor it now 🤣 

But really, you're right, my bad, I didn't notice it was the same thing